bootstrap/kubeadm/main.go

```diff
@@ -164,15 +162,6 @@ func main() {
 
 	// klog.Background will automatically use the right logger.
 	ctrl.SetLogger(klog.Background())
-	if profilerAddress != "" {
```

Comment:
I assume the manager does this for us now? (I see we're passing in the profilerAddress into the options object below)

Reply:
Exactly!